src/material-examples/autocomplete-overview/autocomplete-overview-example.html

```diff
@@ -1,9 +1,21 @@
-<md-input-container>
-  <input mdInput placeholder="State" [mdAutocomplete]="auto" [formControl]="stateCtrl">
-</md-input-container>
+<form class="example-form">
+  <md-input-container class="example-full-width">
+    <input mdInput placeholder="State" [mdAutocomplete]="auto" [formControl]="stateCtrl">
+  </md-input-container>
 
-<md-autocomplete #auto="mdAutocomplete">
-  <md-option *ngFor="let state of filteredStates | async" [value]="state">
-    {{ state }}
-  </md-option>
-</md-autocomplete>
\ No newline at end of file
+  <md-autocomplete #auto="mdAutocomplete">
+    <md-option *ngFor="let state of filteredStates | async" [value]="state.name">
+      <img style="vertical-align:middle;" src="{{state.flag}}" height="25" />
```

Comment:
Probably make this `img` `aria-hidden` since it's a purely decorative addition that doesn't add any semantic meaning to the item.